functions/remediators/remediateGoogleAPIKey/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "gcp-api-key-remediator",
+  "version": "1.0.0",
+  "description": "Lambda which remediates GCP API Keys",
+  "scripts": {
+    "build": "npx tsc"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@tsconfig/node14": "^1.0.1",
+    "@types/aws-lambda": "^8.10.89",
+    "@types/ip": "^1.1.0",
+    "@types/json-schema": "^7.0.9",
+    "@types/node": "^17.0.7",
+    "@typescript-eslint/eslint-plugin": "^5.9.0",
+    "@typescript-eslint/parser": "^5.9.0",
+    "eslint": "^8.6.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "prettier": "^2.5.1",
+    "ts-node": "^10.4.0",
+    "tslib": "^2.3.1",
+    "typescript": "^4.5.4",
+    "axios": "^0.26.1"
+  },
+  "engines": {
+    "node": "16"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/NickLiffen/GSSAR"
+  },
+  "dependencies": {
+    "@aws-sdk/client-iam": "^3.45.0",
```

Comment:
This isn't needed 💯 Once this gets removed we can merge this 👍

Reply:
Kinda forgot about this last requested change :) @NickLiffen Fixed.